src/app/[locale]/_utils/api-utils.ts

```diff
@@ -72,3 +89,43 @@ export const extractResponse = async <R>(
     throw toError(response.left);
   }
 };
+
+export function retryingFetch(): (
+  input: RequestInfo | URL,
+  init?: RequestInit | undefined
+) => Promise<Response> {
+  // a fetch that can be aborted and that gets cancelled after fetchTimeoutMs
+  const abortableFetch = AbortableFetch(agent.getHttpFetch(process.env));
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const timeoutFetch: (input: any, init?: RequestInit | undefined) => Promise<Response> = toFetch(
+    setFetchTimeout(API_TIMEOUT, abortableFetch)
+  );
+  // configure retry logic with default exponential backoff
+  // @see https://github.com/pagopa/io-ts-commons/blob/master/src/backoff.ts
+  const exponentialBackoff = calculateExponentialBackoffInterval();
+  const retryLogic = withRetries<Error, Response>(API_MAX_RETRY, exponentialBackoff);
+  const retryWithError = retryLogicForResponseError((_: Response) => _.status === 500, retryLogic);
```

Comment:
What about passing `exponentialBackoff`, `API_MAX_RETRY` and status code to retry with as arguments defaulting to standard values (the values you are using inside the function)? What do you think about it?